snowdialect.py

```diff
@@ -609,7 +609,7 @@ def get_view_definition(self, connection, view_name, schema=None, **kw):
             ret = cursor.fetchone()
             if ret:
                 return ret[n2i['text']]
-        except Exxception:
```

Comment:
Oh, how did we not catch this until now? O.o

Reply:
maybe no one has ever met with the exception case, I was surprised too